### PR TITLE
docs/moderation: document moderation page

### PR DESCRIPTION
**Pull request by SoggySaussages**
documents the moderation page and initializes the subpages for the
main moderation commands.

I've chosen to document the plugin permissions notice here, and also to call the "log x moderation events not made by YAGPDB" "Foreign logging" which sounds even more dumb now that I'm typing it out, but I don't like "3rd party logging" either, so would appreciate input on this.

**Terms**
- [x] I have read and understood this project's [Contributing Guidelines](CONTRIBUTING.md)

### CODE REVIEW

File: content/moderation/moderation/_index.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
++++
+archetype = "chapter"
+title = "Moderation"
+weight = 1
++++
+
+Everything in moderation, including moderation.
```
Comment (l-zeuch) [inline content/moderation/moderation/_index.md:7]: image

```suggestion
Everything in moderation, including moderation.

<!--more-->
```

File: content/moderation/moderation/_index.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+![Overview of the moderation page](overview_moderation.png)
+
+<center>
+
+**1** Mod Log Channel **2** Report Channel **3** Moderation DM Error Channel **4** Clean Command **5** Report
+Command **6** Unban Logging **7** Giverole/Takeerole commands **8** Give/Takerole Permitted Roles **9**
```
Comment (l-zeuch) [inline content/moderation/moderation/_index.md:14]:
```suggestion
Command **6** Unban Logging **7** Giverole/Takerole commands **8** Give/Takerole Permitted Roles **9**
```

File: content/moderation/moderation/_index.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+<center>
+
+**1** Permitted Roles **2** Moderation DM
+
+</center>
+
+### Permitted Roles
+
+Selecting permitted roles will allow said roles to run the moderation command **EVEN IF** they do not have
```
Comment (l-zeuch) [inline content/moderation/moderation/_index.md:89]: I think bold is just fine, no need to shout ;-)
```suggestion
Selecting permitted roles will allow said roles to run the moderation command **even if** they do not have
```

File: content/moderation/moderation/_index.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+When using the `ban` and `mute` commands with a duration, YAGPDB will essentially execute the inverse commands as itself
+at the end of the duration. i.e, if you execute `ban @user 1d`, YAGPDB will ban the user, and 1 day later YAGPDB will
+execute the `unban` command with reason "Timed ban expired." This action appears in the mod log as YAGPDB being the
+moderator.
+
+Toggling Unban Logging (**6**) allows you to enable or disable the logging of Unmute events.
+
+Toggling Give/Takerole Logging (**9**) allows you to enable or disable the logging of Give/Takerole events.
+
+### Foreign logging
```
Comment (l-zeuch) [inline content/moderation/moderation/_index.md:59]: Shouldn't this be Title Case?
```suggestion
### Foreign Logging
```

**Review by l-zeuch — COMMENTED** [review]
Seeing as this is to be merged into a WIP branch, I'll be a bit lenient
with the requested changes; consider most of them a mental note for
myself, so that I can address them in the future over at #15.

If you do want to address them, feel free to do so, but it's not
*exactly* a blocker.

Biggest question I have, though, is whether we *really* need a sub-page
for every tab on the Moderation page.

I think you were just setting them up to populate them later, however I
think there shouldn't be *that* much to document about them — the
interface is IMO quite intuitive already, so just leaving a few
sentences about each tab should be enough.

File: content/moderation/moderation/_index.md (hunk 3 above)
Comment (SoggySaussages) [reply content/moderation/moderation/_index.md:89]: i was a little too energetic last night :)

**Comment by l-zeuch** [comment]
After careful consideration (read: playing DRG and then going to bed), I
think we can keep the separate pages on each tab for now.

We'll see if they make sense keeping around, once they're populated with
a little more content.

**Pull request merged**